<?xml version="1.0" encoding="UTF-8" ?>
<ui version="4.0">
  <class>DlgRealtimeDlg</class>
  <widget class="QDialog" name="DlgRealtimeDlg">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>627</width>
        <height>451</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Realtime beat detection</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff />
      </iconset>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
        <layout class="QHBoxLayout">
          <item>
            <spacer>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>20</width>
                  <height>26</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="btnOk">
              <property name="text">
                <string>Quit</string>
              </property>
              <property name="shortcut">
                <string>Alt+K</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="0">
        <widget class="QFrame" name="frame">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <widget class="WVuMeter" name="VuRight" native="true">
            <property name="geometry">
              <rect>
                <x>580</x>
                <y>10</y>
                <width>16</width>
                <height>121</height>
              </rect>
            </property>
            <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>10</horstretch>
                <verstretch>100</verstretch>
              </sizepolicy>
            </property>
          </widget>
          <widget class="WVuMeter" name="VuLeft" native="true">
            <property name="geometry">
              <rect>
                <x>560</x>
                <y>10</y>
                <width>16</width>
                <height>121</height>
              </rect>
            </property>
            <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>10</horstretch>
                <verstretch>100</verstretch>
              </sizepolicy>
            </property>
          </widget>
          <widget class="WWaveform" name="waveform" native="true">
            <property name="geometry">
              <rect>
                <x>10</x>
                <y>280</y>
                <width>591</width>
                <height>51</height>
              </rect>
            </property>
          </widget>
          <widget class="WEnergyBeatDisplay" name="beatDisplay" native="true">
            <property name="geometry">
              <rect>
                <x>490</x>
                <y>10</y>
                <width>61</width>
                <height>121</height>
              </rect>
            </property>
          </widget>
          <widget class="QFrame" name="frmMagnitude">
            <property name="geometry">
              <rect>
                <x>240</x>
                <y>10</y>
                <width>241</width>
                <height>121</height>
              </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>0</number>
              </property>
              <item row="0" column="0">
                <widget class="WPlot" name="magDisplay" native="true" />
              </item>
            </layout>
          </widget>
          <widget class="QFrame" name="frmCounter">
            <property name="geometry">
              <rect>
                <x>10</x>
                <y>10</y>
                <width>221</width>
                <height>261</height>
              </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
            </property>
            <widget class="QLCDNumber" name="lcdCurrentBPM">
              <property name="geometry">
                <rect>
                  <x>100</x>
                  <y>10</y>
                  <width>111</width>
                  <height>41</height>
                </rect>
              </property>
              <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
              </property>
              <property name="smallDecimalPoint">
                <bool>false</bool>
              </property>
              <property name="digitCount">
                <number>6</number>
              </property>
              <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
              </property>
            </widget>
            <widget class="QLabel" name="txtBpm">
              <property name="geometry">
                <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>71</width>
                  <height>41</height>
                </rect>
              </property>
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>BPM:</string>
              </property>
            </widget>
            <widget class="QFrame" name="frmManualCounter">
              <property name="geometry">
                <rect>
                  <x>10</x>
                  <y>140</y>
                  <width>201</width>
                  <height>111</height>
                </rect>
              </property>
              <property name="frameShape">
                <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
              </property>
              <widget class="QPushButton" name="btnReset">
                <property name="geometry">
                  <rect>
                    <x>10</x>
                    <y>80</y>
                    <width>51</width>
                    <height>20</height>
                  </rect>
                </property>
                <property name="font">
                  <font>
                    <pointsize>8</pointsize>
                  </font>
                </property>
                <property name="text">
                  <string>Reset</string>
                </property>
              </widget>
              <widget class="QLabel" name="lblBeatsDisp">
                <property name="geometry">
                  <rect>
                    <x>100</x>
                    <y>53</y>
                    <width>57</width>
                    <height>17</height>
                  </rect>
                </property>
                <property name="text">
                  <string>0</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
              <widget class="QPushButton" name="btnBeat">
                <property name="geometry">
                  <rect>
                    <x>110</x>
                    <y>80</y>
                    <width>51</width>
                    <height>20</height>
                  </rect>
                </property>
                <property name="font">
                  <font>
                    <pointsize>8</pointsize>
                  </font>
                </property>
                <property name="text">
                  <string>Beat</string>
                </property>
                <property name="default">
                  <bool>true</bool>
                </property>
              </widget>
              <widget class="QPushButton" name="btnSync">
                <property name="geometry">
                  <rect>
                    <x>60</x>
                    <y>80</y>
                    <width>51</width>
                    <height>20</height>
                  </rect>
                </property>
                <property name="font">
                  <font>
                    <pointsize>8</pointsize>
                  </font>
                </property>
                <property name="text">
                  <string>Sync</string>
                </property>
              </widget>
              <widget class="QLabel" name="lblErrorDisp">
                <property name="geometry">
                  <rect>
                    <x>100</x>
                    <y>33</y>
                    <width>57</width>
                    <height>17</height>
                  </rect>
                </property>
                <property name="text">
                  <string>0</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
              <widget class="QProgressBar" name="pbInterval">
                <property name="geometry">
                  <rect>
                    <x>170</x>
                    <y>3</y>
                    <width>16</width>
                    <height>101</height>
                  </rect>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>16</width>
                    <height>16777215</height>
                  </size>
                </property>
                <property name="value">
                  <number>0</number>
                </property>
                <property name="textVisible">
                  <bool>false</bool>
                </property>
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
              </widget>
              <widget class="QLabel" name="lblManBpm">
                <property name="geometry">
                  <rect>
                    <x>10</x>
                    <y>0</y>
                    <width>57</width>
                    <height>31</height>
                  </rect>
                </property>
                <property name="text">
                  <string>BPM:</string>
                </property>
              </widget>
              <widget class="QLCDNumber" name="lcdTapBPM">
                <property name="geometry">
                  <rect>
                    <x>80</x>
                    <y>0</y>
                    <width>81</width>
                    <height>31</height>
                  </rect>
                </property>
                <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                </property>
                <property name="smallDecimalPoint">
                  <bool>false</bool>
                </property>
                <property name="digitCount">
                  <number>6</number>
                </property>
                <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                </property>
              </widget>
              <widget class="QLabel" name="lblManBeats">
                <property name="geometry">
                  <rect>
                    <x>10</x>
                    <y>53</y>
                    <width>57</width>
                    <height>17</height>
                  </rect>
                </property>
                <property name="text">
                  <string>Beats:</string>
                </property>
              </widget>
              <widget class="QLabel" name="lblManError">
                <property name="geometry">
                  <rect>
                    <x>10</x>
                    <y>33</y>
                    <width>57</width>
                    <height>17</height>
                  </rect>
                </property>
                <property name="text">
                  <string>Error:</string>
                </property>
              </widget>
            </widget>
            <widget class="QLabel" name="txtCurrentBPM">
              <property name="geometry">
                <rect>
                  <x>10</x>
                  <y>50</y>
                  <width>121</width>
                  <height>21</height>
                </rect>
              </property>
              <property name="text">
                <string>Current BPM:</string>
              </property>
            </widget>
            <widget class="QLabel" name="lblCurrentBPM">
              <property name="geometry">
                <rect>
                  <x>120</x>
                  <y>50</y>
                  <width>91</width>
                  <height>21</height>
                </rect>
              </property>
              <property name="text">
                <string>0.0</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
            </widget>
            <widget class="QLabel" name="lblError">
              <property name="geometry">
                <rect>
                  <x>120</x>
                  <y>70</y>
                  <width>91</width>
                  <height>21</height>
                </rect>
              </property>
              <property name="text">
                <string>0.00</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
            </widget>
            <widget class="QLabel" name="txtError">
              <property name="geometry">
                <rect>
                  <x>10</x>
                  <y>70</y>
                  <width>121</width>
                  <height>21</height>
                </rect>
              </property>
              <property name="text">
                <string>Error:</string>
              </property>
            </widget>
            <widget class="QPushButton" name="button">
              <property name="enabled">
                <bool>false</bool>
              </property>
              <property name="geometry">
                <rect>
                  <x>170</x>
                  <y>100</y>
                  <width>41</width>
                  <height>31</height>
                </rect>
              </property>
              <property name="text">
                <string />
              </property>
              <property name="checkable">
                <bool>true</bool>
              </property>
              <property name="checked">
                <bool>false</bool>
              </property>
            </widget>
          </widget>
          <widget class="WWaveform" name="calcWave" native="true">
            <property name="geometry">
              <rect>
                <x>10</x>
                <y>340</y>
                <width>591</width>
                <height>51</height>
              </rect>
            </property>
          </widget>
          <widget class="WBPMCalcDisplay" name="xcorrDisplay" native="true">
            <property name="geometry">
              <rect>
                <x>240</x>
                <y>140</y>
                <width>361</width>
                <height>131</height>
              </rect>
            </property>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <customwidgets>
    <customwidget>
      <class>WVuMeter</class>
      <extends>QWidget</extends>
      <header>wvumeter.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>WWaveform</class>
      <extends>QWidget</extends>
      <header>wwaveform.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>WEnergyBeatDisplay</class>
      <extends>QWidget</extends>
      <header>wenergybeatdisplay.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>WPlot</class>
      <extends>QWidget</extends>
      <header>wplot.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>WBPMCalcDisplay</class>
      <extends>QWidget</extends>
      <header>wbpmcalcdisplay.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local">progressbar.h</include>
  </includes>
  <resources />
  <connections>
    <connection>
      <sender>btnOk</sender>
      <signal>clicked()</signal>
      <receiver>DlgRealtimeDlg</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>20</x>
          <y>20</y>
        </hint>
        <hint type="destinationlabel">
          <x>20</x>
          <y>20</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
